app/models/move.rb

```diff
@@ -42,7 +42,7 @@ class Move < VersionedModel
   has_many :notifications, as: :topic, dependent: :destroy # NB: polymorphic association
   has_many :journeys, -> { default_order }, dependent: :restrict_with_exception, inverse_of: :move
   has_many :court_hearings, dependent: :restrict_with_exception
-  has_many :events, as: :eventable, dependent: :destroy # NB: polymorphic association
+  has_many :events, as: :eventable, dependent: :destroy, class_name: 'MoveEvent' # NB: polymorphic association
```

Comment:
If this works I think it's slightly cleaner and more explicit (guessing here)...

```ruby
has_many :move_events, as: :eventable, dependent: :destroy
```

Reply:
👍